4-WebApp-your-API/4-1-MyOrg/README.md

```diff
@@ -427,8 +478,17 @@ In the left-hand navigation pane, select the **Azure Active Directory** service,
 > NOTE: Remember, the To Do list is stored in memory in this TodoListService sample. Azure Web Sites will spin down your web site if it is inactive, and your To Do list will get emptied.
 Also, if you increase the instance count of the web site, requests will be distributed among the instances. To Do will, therefore, not be the same on each instance.
 
+</details>
+
+## Next Steps
 
-## More information
+- Learn how to use app groups. [Add authorization using security groups & groups claims to a Web app thats signs-in users with the Microsoft identity platform](../../5-WebApp-AuthZ/5-2-Groups/README.md).
+ 
+## Learn More
+
+
+<details>
+ <summary>Expand the section</summary>
```

Comment:
Lets not hide this section, is reraely visited as is and we want to highlight theese